build-release

```diff
@@ -35,13 +35,13 @@ throwError () {
 
 	case "${error_keyword}" in
 		'BADREQUEST')
-			error_code='400'
+			error_code=2
 			;;
 		'INTERNAL')
-			error_code='500'
+			error_code=3
 			;;
 		'NOTIMPLEMENTED')
-			error_code='501'
+			error_code=4
```

Comment:
I liked my HTTP-like error codes, at least the numbers had a meaning 😁

Reply:
You can't actually return them... the value gets taken modulo 256, so if you `exit 501` then `$?` equals 245.

Reply:
Oh… 😢